GPU/GPUTracking/TRDTracking/GPUTRDTracker.cxx

```diff
@@ -657,6 +657,22 @@ GPUd() bool GPUTRDTracker_t<TRDTRK, PROP>::FollowProlongation(PROP* prop, TRDTRK
             GPUWarning("Track parameter for track %i, x=%f at chamber %i x=%f in layer %i cannot be retrieved", iTrk, trkWork->getX(), currDet, mR[currDet], iLayer);
           }
         }
+        // check if track crosses padrows
+        if (iDet == 0) {
+          float projZEntry, projYEntry;
+          // Get Z for Entry of Track
+          prop->getPropagatedYZ(trkWork->getX() - 3.f, projYEntry, projZEntry);
```

Comment:
The drift length of 3cm you should be able to find in the geometry

Reply:
Ok.